applications/DeepSpeed-Chat/training/utils/model/model_utils.py

```diff
@@ -130,7 +130,9 @@ def create_critic_model(model_name_or_path,
     critic_model = create_hf_model(AutoModel, model_name_or_path, tokenizer,
                                    ds_config, rlhf_training, dropout)
     end = time.time()
-    if torch.distributed.get_rank() == 0:
+    if not torch.distributed.is_initialized():
+        print(f"> Creating model from_config took {end - start} seconds")
+    elif torch.distributed.get_rank() == 0:
```

Comment:
Rather than repeat this code over and over again, could we copy the `print_rank_0` code from here?
https://github.com/microsoft/DeepSpeedExamples/blob/master/applications/DeepSpeed-VisualChat/utils/utils.py#L15

We can replace the current `print_rank_0` that we have here:
https://github.com/microsoft/DeepSpeedExamples/blob/master/applications/DeepSpeed-Chat/training/utils/utils.py#L17

Then it just becomes `print_rank_0(f"> Creating model from_config took {end - start} seconds")`

Reply:
Great idea. Just replaced it and confirmed it is working